### ISSUE TITLE
Minor issue with autorrect for add_index

### ISSUE THREAD

**Issue by kathgironpe** [r7kamura/rubocop-migration#7]
Hi!

Thanks for the gem. 

I found a few issues and this is the most critical one: 

When a migration tries to use `add_index` multiple times, the line `disable_ddl_transaction!` is added multiple times:

```ruby
  disable_ddl_transaction!

  disable_ddl_transaction!

  disable_ddl_transaction!

  disable_ddl_transaction!

...
```

**Comment by r7kamura** [comment]
Thank you @kathgironpe for reporting!

I was able to reproduce this problem at the following pull request:

- https://github.com/r7kamura/rubocop-migration/pull/8

The solution would be to prepare a separate cop that removes duplicated `disable_ddl_transaction!`, or use a mechanism to prevent such autocorrection in RuboCop, if there is one.

**Comment by kathgironpe** [comment]
@r7kamura I also found other issues but have to confirm them with tests. 

I believe there is a way to avoid this for sure.

Try if asking for the definition for `disable_ddl_transaction!` exists.

In this file, we can check if a method named change exists... 

https://github.com/rubocop/rubocop-rails/blob/master/lib/rubocop/cop/rails/reversible_migration.rb

**Comment by r7kamura** [comment]
Just released v0.4.1 ✨💎✨ 

- https://rubygems.org/gems/rubocop-migration/versions/0.4.1
- https://github.com/r7kamura/rubocop-migration/releases/tag/v0.4.1
